identity/client/src/pages/setup/styled.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import styled from "styled-components";
+import {
+  Stack,
+  InlineNotification as BaseInlineNotification,
+} from "@carbon/react";
+import { spacing06, spacing07 } from "@carbon/elements";
+import Page from "src/components/layout/Page.tsx";
+
+const setupPageContentWidth = "25rem";
+
+export const SetupFormContainer = styled(Stack)<{ $hasError: boolean }>`
+  flex-direction: column;
+  gap: ${spacing06};
+  width: 100%;
+  padding-top: ${({ $hasError }) => ($hasError ? 0 : spacing06)};
+
+  > button {
+    max-inline-size: unset;
+    width: 100%;
+  }
```

Comment:
Why are we overwriting the button styles? The Carbon button component should already have all the proper styling and overwriting goes against the design system

Reply:
I'm not sure, I based it off the form for the login page, but without any css the button is not set at 100% of the size of the form. Same is true for login page in other components, it's just set differently, for example - [Operate's login page submit button](https://github.com/camunda/camunda/blob/9e911d28da248ad953914c4582850a2bfa970410/operate/client/src/App/Login/styled.ts#L16). Same for tasklist. What I'll do is match this styling to the other components using `min-width`.